packages/synapse-react-client/src/components/StorybookComponentWrapper.tsx

```diff
@@ -54,6 +55,10 @@ function overrideEndpoint(stack: SynapseStack) {
   ;(window as any)['SRC_OVERRIDE_ENDPOINT_CONFIG'] = endpointConfig
 }
 
+// Store auth token in global atom to avoid race conditions where the Story component
+// may not get the login token right away
+const authTokenAtom = atom<string | undefined>(undefined)
```

Comment:
I think Storybook may have changed something that caused this wrapper component to be completely re-created instead of reused when opening a new story, so `accessToken` would always be `undefined` until it was retrieved. This would cause stories requiring authentication to tell you to log in even if you already did. This should fix that.